apps/impala/src/impala/api.py

```diff
@@ -44,8 +44,10 @@
 
 if sys.version_info[0] > 2:
   from django.utils.translation import gettext as _
+  from base64 import decodebytes
 else:
   from django.utils.translation import ugettext as _
+  from base64 import decodestring as decodebytes
```

Comment:
I would suggest to alias them here as `base64_decodebytes`
Like this:

```python
if sys.version_info[0] > 2:
  from base64 import decodebytes as base64_decodebytes
else:
  from base64 import decodestring as base64_decodebytes
```

My thinking is that this way in later part of code we will instantly know from where this decodebytes is coming from but would love to hear other views on this. cc: @amitsrivastava @wing2fly